api/core/model_runtime/model_providers/openllm/llm/openllm_generate.py

```diff
@@ -106,7 +106,7 @@ def generate(
             timeout = 120
 
         data = {
-            "stop": stop or [],
+            "stop": stop,
```

Comment:
There might be a type hint error for the `stop` parameter here. In the caller, the type hint for the `stop` parameter is:

```python
    def _generate(
        self,
        model: str,
        credentials: dict,
        prompt_messages: list[PromptMessage],
        model_parameters: dict,
        tools: list[PromptMessageTool] | None = None,
        stop: list[str] | None = None,
        stream: bool = True,
        user: str | None = None,
    ) -> LLMResult | Generator:
        ...
```

Would you mind fixing the type hint for the `stop` parameter?

Reply:
yes will do that, thanks